resources/views/auth/application/files.blade.php

```diff
@@ -11,7 +11,12 @@
             <blockquote>
                 A pályázatnak az alábbiakat kell tartalmaznia:
                 <ul style="margin-left:20px;margin-top:0">
-                    <li style="list-style-type: circle !important">önéletrajz (hagyományos, leíró jellegű)</li>
+                    <li style="list-style-type: circle !important"><em>hagyományos, leíró jellegű</em> önéletrajz
+                        <i class="material-icons tooltipped" style="font-size: 1em; vertical-align: -0.2em; cursor: default"
+                           data-tooltip="Teljes mondatokból álló, összefüggő szöveg (NEM Europass-jellegű)">
```

Comment:
My suggestion would be this:

`data-tooltip="teljes mondatokból álló, összefüggő szöveg (**nem** Europass-jellegű)">`

This way, it would pass more into the context (which does not use capital letters) and could look a bit more professional.

It would be even better to have `nem` with a bold font if that could be solved. But I can live without that.

Reply:
`data-tooltip` only takes plain text, unfortunately.

Agree on the capitalization.

Reply:
OK:) then let's stay with the capitalization.